crt_portal/cts_forms/views.py

```diff
@@ -18,11 +18,18 @@ def IndexView(request):
     # formatting protected class
     for report in latest_reports:
         reports = []
-        for p_class in report.protected_class.all().order_by('-form_order'):
+        for p_class in report.protected_class.all().order_by('form_order'):
             if p_class.protected_class is not None:
-                reports.append(PROTECTED_CLASS_CODES.get(p_class.protected_class, p_class.protected_class))
+                code = PROTECTED_CLASS_CODES.get(p_class.protected_class, p_class.protected_class)
+                if code is not None:
```

Comment:
Code should never be `None`, because line 23 specifies that if no value for is found in the `PROTECTED_CLASS_CODES`, the longer protected class name will be supplied as a default. Guarding against null values is good, but two guards back-to-back seems like too many. Also, we never handle the `if code is None` branch — I feel like if we are going to check for `code` being `None`, we should handle the possibility that `code is None`.